talk/morelanguage/move.tex

```diff
@@ -205,24 +208,47 @@
     \begin{cppcode*}{}
       class Movable {
         Movable();
-        Movable(Movable &&other) :
+        Movable(const Movable &other);
+        Movable(Movable &&other) noexcept :
           Movable() {         // constructor delegation
           swap(*this, other);
         }
-        Movable& operator=(Movable other) { // by value
+        Movable& operator=(Movable other) noexcept { // by value
           swap(*this, other);
           return *this;
         }
-        friend void swap(Movable &a, Movable &b);
+        friend void swap(Movable &a, Movable &b) noexcept {...}
       };
-      void swap(Movable &a, Movable &b);
       Movable a, b;
       a = b;            // operator= copies b into "other"
       a = std::move(b); // operator= moves b into "other"
     \end{cppcode*}
   \end{exampleblock}
 \end{frame}
 
+\begin{frame}[fragile]
+  \frametitlecpp[11]{Move semantics : the complete picture}
```

Comment:
```suggestion
  \frametitlecpp[11]{Move semantics: the complete picture}
```

Reply:
I will not mess with French typesetting, if it is used consistently in the presentation.